backend/src/main/java/com/greenfoxacademy/backend/dtos/LoginUserDto.java

```diff
@@ -0,0 +1,17 @@
+package com.greenfoxacademy.backend.dtos;
+
+import com.greenfoxacademy.backend.services.ValidPassword;
+import jakarta.validation.constraints.Email;
+import lombok.Data;
+
+
+/**
+ * Data Transfer Object for Login user.
+ */
+@Data
+public class LoginUserDto {
+  @Email
+  private String email;
+  @ValidPassword
+  private String password;
+}
```

Comment:
If a class is that small and should not be modifiable, you can just create it as a record.